rules/autoconsent/openli.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "openli",
+  "vendorUrl": "https://openli.com",
+  "prehideSelectors": [".legalmonster-cleanslate"],
+  "detectCmp": [
+    {
+      "exists": ".legalmonster-cleanslate"
+    }
+  ],
+  "detectPopup": [
+    {
+      "visible": ".legalmonster-cleanslate",
```

Comment:
This seems to be too permissive, and leading to the rule triggering on every page load. On chargebee.com, this matches the shield in the bottom left corner which is always visible:
![Screenshot 2024-02-01 at 13 21 42](https://github.com/duckduckgo/autoconsent/assets/248111/9b3eda34-9e7f-4574-9964-e5cda2ceef23)

Reply:
Thanks for catching this! I made the selectors more specific